### DISCUSSION TITLE
请问一下,Visual studio如何设置括号对的颜色呀,我用的是浅色主题

### DISCUSSION THREAD

**Discussion by oufu99** [BYJRK/DotNet-Discussions#175] [C# 技术交流]
在网上看到别人的括号是有颜色的,我要怎么设置呀~
![image](https://github.com/BYJRK/DotNet-Discussions/assets/18545756/b319988a-7c13-4559-9911-bda8f200d03e)

我现在自己的
![image](https://github.com/BYJRK/DotNet-Discussions/assets/18545756/d3ce1f69-147a-4088-85a0-58f8611570fb)

**Comment by BYJRK** [comment]
这个是 2022 比较新版本原生自带的功能。如果你在旧版本，可以试试一些插件，比如：

[RainbowBraces](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.RainbowBraces)

**Reply by oufu99** [reply]
用了这个插件~,谢谢
